test/surface_test.py

```diff
@@ -1580,7 +1580,7 @@ def test_set_alpha_none(self):
         s = pygame.Surface((1, 1), SRCALPHA, 32)
         s.fill((0, 255, 0, 128))
         s.set_alpha(None)
-        self.assertEqual(None, s.get_alpha())
+        self.assertEqual(255, s.get_alpha())
```

Comment:
set_alpha(None) used to make get_alpha() to return None.

Should we keep this for compatibility? I'm not sure.

If we keep the new behaviour, we should have a separate test for SDL1 or fix SDL1 to do the same?